src/layouts/ReviewRequest/components/Comments/SendCommentForm.tsx

```diff
@@ -39,6 +44,8 @@ export const SendCommentForm = ({
 
   const handleUpdateNotifications = async ({ comment }: CommentFormProps) => {
     await updateNotifications({ siteName, requestId, message: comment })
+    await refetchData()
```

Comment:
actually if we just want the data to be refetched on data update, we could just pass `CommentData` and invalidate its key on data refetch. this might be slightly clearer also

Reply:
You're right, invalidating the query is much simpler, I've implemented the suggestion in https://github.com/isomerpages/isomercms-frontend/pull/1133/commits/38bd4010d56b9540ee4799e69569899935692c63

Reply:
should be done in hook but it's fine for now!